razorpay/script.js

```diff
@@ -115,6 +117,13 @@ document.addEventListener('DOMContentLoaded', function(event) {
   });
 
 
+  var loadingimage =  function(defaults) {
```

Comment:
correct the function name, 
It should be `fullscreenLoader`,  It should have option to start and stop.